lib/log.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const removeSensitiveData = (definition, instance) => {
+  const filtered = {};
+  for (const name of Object.keys(instance)) {
+    const prop = definition[name];
+    if (
```

Comment:
```suggestion
    if (prop) {
      if (prop.sensitive) continue;
      if (prop.definition && prop.definition.sensitive) continue;
    }
```

Reply:
+ remove next 3 lines and put `filtered[name] = instance[name];` after if statement